importer/SeBbrSv.py

```diff
@@ -141,9 +139,11 @@ def set_no_of_buildings(self):
         extracted_no = utils.get_number_from_string(
             utils.get_text_inside_brackets(self.funktion))
         if extracted_no is not None:
+            ref = self.wlm_source
             self.add_statement(
                 "has_parts_of_class", "Q41176",
-                {"quantity": {"quantity_value": extracted_no}})
+                {"quantity": {"quantity_value": extracted_no}},
+                refs=[ref])
```

Comment:
And taking this one step further. Would it be possible to use `self.wlm_source` as a default?
I'm thinking something like:
```
def add_statement(..., refs=None, ...):
    if refs is None or refs is True:
        refs = self.wlm_source
    elif refs is False:
        refs = None
    ...
```
That way you can still opt not to use a ref (e.g. for images) using `refs=False` but by default it would use `refs=self.wlm_source`. Or you can make `refs=True` trigger the default behaviour and require that `refs` always takes a value (It would then allow True/False, a ref or a list of refs).